scraper/ordoclic.py

```diff
@@ -0,0 +1,85 @@
+import httpx
+from httpx import TimeoutException
+import json
+from datetime import datetime, timedelta
+
+timeout = httpx.Timeout(15.0, connect=15.0)
+session = httpx.Client(timeout=timeout)
+
+# get all slugs
+def search():
+    base_url = 'https://api.ordoclic.fr/v1/public/search'
+    payload = {'page': '1', 'per_page': '500', 'in.isCovidVaccineSupported': 'true', 'in.isPublicProfile': 'true' }
```

Comment:
J'aurai mis direct les 10k si c'est bien la limite haute, ou meme mieux deja implementer le crawl de toute les pages avec un for / if pour etre sur qu'au moment ou y aura plus de lieu on ne les loupe pas 😃